provisioning/transport/http/src/Microsoft.Azure.Devices.Provisioning.Transport.Http.csproj

```diff
@@ -20,7 +20,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <Version>1.2.2</Version>
+    <Version>1.2.3</Version>
```

Comment:
why is this patch update? Should it not be minor?

Reply:
It is patch because the fixes for making it 2.1 compatible were minor and mostly bug fixes.

Reply:
But I see other clients updating to minor version for making this compatibility change?

Reply:
Based on changes to ns2.1 in the files, if it is small fix I updated it to reflect patch. If those were big changes then I updated it to reflect minor.